Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -294,6 +300,8 @@ void HTMLModelElement::createModelPlayer()
     ASSERT(document().page());
 #if ENABLE(MODEL_PROCESS)
     m_entityTransform = DOMMatrixReadOnly::create(TransformationMatrix::identity, DOMMatrixReadOnly::Is2D::No);
+    m_boundingBoxCenter = DOMPointReadOnly::create(0, 0, 0, 1);
```

Comment:
this is equivalent to:
```
DOMPointReadOnly::create({ })
``
as DOMPointInit is defined as:
```
struct DOMPointInit {
    double x { 0 };
    double y { 0 };
    double z { 0 };
    double w { 1 };
};
```